app/pathogen/arbovirus/dashboard/(visualizations)/visualization-header.tsx

```diff
@@ -57,7 +57,7 @@ interface AllButtonConfigurations {
 const DownloadButton = (props: DownloadButtonProps) => (
   <button
     id={props.configuration.id}
-    className="mr-4"
+    className="mr-2 p-2 hover:bg-gray-100 rounded-full"
```

Comment:
This should probably use some kind of child selector in the parent, something to probably clean up later :)